### ISSUE TITLE
Training epochs

### ISSUE THREAD

**Issue by kktsubota** [Googolxx/STF#3]
Thank you for sharing the codes of your great work.

I have a question about the training epochs that you used for your paper and for pre-trained models.
In your paper, the number of iterations is 1.8M, and your model sees 1.8M * 16 = 28.8M images in total. However, in your README, the number of epochs is 1000, and your model sees 0.3M (the size of the subset of OpenImagesV6) * 1000 = 300M images in total.
The number is not equivalent, so I would appreciate it if you could tell me how did you train your model for your paper and the pre-trained models.

**Comment by Googolxx** [comment]
I stop training at 100th epochs roughly,  set learning rate = 1e-4 for 60~80 epochs (depends on whether loss keep converge), and set learning rate = 1e-5 for last 20 epoch.
And you can try setting learning rate = 1e-6 for 1~2 epochs after you finish the training above, which may provide a boost of 5e-3 PSNR or less.

**Comment by Googolxx** [comment]
I stop training at 100th epochs roughly, set learning rate = 1e-4 for 60~80 epochs (depends on whether the loss keep converge), and set learning rate = 1e-5 for last 20 epoch.
And you can try setting learning rate = 1e-6 for 12 epochs after you finish the training above, which may provide a boost of 5e-3 PSNR or less.

**Comment by Googolxx** [comment]
I stop training at 100th epochs roughly, set learning rate = 1e-4 for 60~80 epochs (depends on whether the loss keep converge), and set learning rate = 1e-5 for last 20 epoch.
And you can try setting learning rate = 1e-6 for 1~2 epochs after you finish the training above, which may provide a boost of 5e-3 PSNR or less.

**Comment by Googolxx** [comment]
I stop training at 100th epochs roughly, set learning rate = 1e-4 for 60-80 epochs (depends on whether the loss keep converge), and set learning rate = 1e-5 for last 20 epoch.
And you can try setting learning rate = 1e-6 for 1-2 epochs after you finish the training above, which may provide a boost of 5e-3 PSNR or less.

**Comment by kktsubota** [comment]
Thank you for your prompt reply. I will check it.
